orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -139,15 +116,15 @@ void GenerateRandomDataWithOneHot(
       int64_t D = input_shapes[i][input_shapes[i].NumDimensions() - 1];
 
       std::fill(x_datas[i].begin(), x_datas[i].end(), (T)0);
-      for (int64_t k = 0; k < N; k++)
-        x_datas[i][k * D + (seed % D)] = (T)1;
+      for (int64_t k = 0; k < N; k++) x_datas[i][k * D + (seed % D)] = (T)1;
     } else {
       std::generate(x_datas[i].begin(), x_datas[i].end(), [&] { return distribution(generator); });
     }
   }
 }
 
-void UnaryOpGradientTest(const std::string& op_type, const std::string& domain = kOnnxDomain, const int opset_version = 9,
+void UnaryOpGradientTest(const std::string& op_type, const std::string& domain = kOnnxDomain,
+                         const int opset_version = 9,
```

Comment:
does this mean we are testing version 9 for every op? Should we move this to latest? This can be done in a different PR.

Reply:
Right. I think most of the UTs in ORT (not just gradient checker, but also Op tester) just have a default OpSet version, and since it's not modified for long time, it's very small. The new UTs added need to specify higher OpSet version manually if it wants to test the new version. We may need to modify lots of such numbers to update our UTs.